tailwind.config.js

```diff
@@ -1,11 +1,9 @@
 const plugin = require("tailwindcss/plugin");
 
 module.exports = {
-  future: {
-    removeDeprecatedGapUtilities: true,
-    purgeLayersByDefault: true,
-  },
   purge: {
+    mode: 'layers',
+    layers: ['utilities'],
```

Comment:
https://tailwindcss.com/docs/optimizing-for-production#purging-specific-layers

> By default, Tailwind will purge all styles in the base, components, and utilities layers.

das ist genau das was wir wollen für den production build. alles weg was nicht gebraucht wird an css.  
können hier beim default bleiben. würde hier nichts anders konfigurieren.

Reply:
also einfach die layers: weg?